packages/test/mocha-test-setup/mocharc-common.js

```diff
@@ -14,8 +14,8 @@ function getFluidTestMochaConfig(packageDir, additionalRequiredModules) {
     const moduleDir = `${packageDir}/node_modules`;
 
     const requiredModules = [
+        ...(additionalRequiredModules ? additionalRequiredModules : []),
         `@fluidframework/mocha-test-setup`, // General mocha setup e.g. suppresses console.log
```

Comment:
can you add comments about why it is important the mocha-test-setup is last in required modules